why-graphql-advantages-disadvantages-alternatives.md

```diff
@@ -39,15 +39,15 @@ GET https://api.domain.com/authors/7
 
-Facebook open sourced the GraphQL specification and its reference implementation in JavaScript. Since then multiple other major programming languages implemented the specification. In addition, the ecosystem around GraphQL is not only growing horizontally by offering multiple programming languages, but also vertically by having libraries on top of GraphQL (e.g. Apollo, Relay).
+Facebook open-sourced the GraphQL specification and its reference implementation in JavaScript, and multiple major programming languages implemented the specification have followed since then. The ecosystem around GraphQL is growing horizontally by offering multiple programming languages, but also vertically, with libraries on top of GraphQL like Apollo and Relay.
 
-A GraphQL operation is either a query (read), mutation (write) or subscription (continuous read). Each of those operations is only a string which needs to be constructed accordingly to the GraphQL query language specification. Once such a GraphQL operation reaches the backend application coming from a frontend application, it can be interpreted against the entire GraphQL schema on the backend and resolved with data for the frontend application. GraphQL is not opinionated about the network layer (which often happens to be HTTP) and not opinionated about the payload format (which often happens to be JSON). It isn't opinionated about the application architecture at all (which often happens to be a frontend/backend architecture). It is only a query language.
+A GraphQL operation is either a query (read), mutation (write), or subscription (continuous read). Each of those operations is only a string that needs to be constructed according to the GraphQL query language specification. Once this GraphQL operation reaches the backend application, it can be interpreted against the entire GraphQL schema there, and resolved with data for the frontend application. GraphQL is not opinionated about the network layer, which is often HTTP, nor about the payload format, which is usually  JSON. It isn't opinionated about the application architecture at all. It is only a query language.
```

Comment:
"A GraphQL operation is either a query (read), mutation (write), or subscription (continuous read)." -> "A GraphQL operation can be either a query (read), mutation (write), or subscription (continuous read). But that's not all to it. The query language is evolving all the time so there may be other operations in the future."